gateway/handle.go

```diff
@@ -670,6 +670,17 @@ func (gw *Handle) internalBatchHandlerFunc() http.HandlerFunc {
 				stats.CountType,
 				gw.newReqTypeStatsTagsWithReason(reqType, ""),
 			).Count(len(jobs))
+
+			// Sending events to config backend
+			for _, job := range jobs {
+				sourceID := gjson.GetBytes(job.Parameters, "source_id").String()
+				sourceConfig := gw.getSourceConfigFromSourceID(sourceID)
+				if sourceConfig.WriteKey == "" {
+					gw.logger.Errorn("unable to get writeKey for job", logger.NewStringField("uuid", job.UUID.String()))
```

Comment:
Shouldn't it be a warning log? Also, Can we add sourceID as well for debugging?
```suggestion
					gw.logger.Warnn("unable to get writeKey for job",
						obskit.SourceID(sourceID),
						logger.NewStringField("jobUUID", job.UUID.String()),
					)
```